samples/matter/smoke_co_alarm/Kconfig

```diff
@@ -22,6 +22,14 @@ config CHIP_ICD_LIT_SUPPORT
 config CHIP_ICD_SIT_SLOW_POLL_LIMIT
 	default 5000
 
+# Use the same value as CONFIG_CHIP_ICD_IDLE_MODE_DURATION
+config CHIP_ICD_SLOW_POLL_INTERVAL
+	default 300000
+
+# Use greater than default (200 ms) value to limit the impact of Active Threshold Duration on power consumption
+config CHIP_ICD_FAST_POLLING_INTERVAL
+	default 500
```

Comment:
why are you setting defaults in a Kconfig file? These should be in a prj.conf file...

Reply:
The overall idea was to keep the configs common to all variants (in case of having multiple prj.confs) in Kconfig, to avoid copying it across multiple files. I can move this one now, but please note that there are already other configs configured in the same way, so it will result in inconsistency and I would not like to refactor all samples in scope of this PR.